.github/workflows/dist.yml

```diff
@@ -67,8 +67,6 @@ jobs:
       matrix:
         os: [windows-latest, macos-latest, ubuntu-20.04]
         python_version:
-        - '3.6'
-        - '3.7'
```

Comment:
`pybind11-stubgen>=1.2` supports Python 3.7. 
3.6 would require me to get rid of `from __future__ import annotations`, which would cause too much of a burden to my code.